### ISSUE TITLE
user guides: Section on best practices

### ISSUE THREAD

**Issue by lilic** [prometheus-operator/website#1]
an idea I have is it would be great to include a best practices section and it can also be used as something to link to our users when they are just starting out.
 
We could include things like:
- unbound high cardinality
-  when to use ServiceMonitor vs PodMonitor
-  symptom vs cause-based Alerting
- etc.

**Comment by metalmatze** [comment]
IMHO these are topics mostly relevant for the Prometheus Operator and should simply live in that repository.
Only if something doesn't belong in either repository docs, then we put it here, I'd say.

For example, list of talks and videos.

**Comment by lilic** [comment]
Would you want them to land on the webpage in the end?

Happy to do either, we can always transfer issue to Prometheus operator repo.

**Comment by metalmatze** [comment]
Yeah, I think it should be in the Prometheus Operator repository and then simply pulled in by the website during build-time.
